packages/client/src/generation/TSClient/utils/invalidImportWarning.ts

```diff
@@ -0,0 +1,5 @@
+export const invalidImportWarning = [
+  'Add your client as a `db` dependency to `package.json` and import it as a dependency instead',
```

Comment:
Is "`db` dependency" universally true?

Reply:
You could technically call it what you want, I am trying to set some convention that is closest to what I have seen so far from our users.

Reply:
Maybe an example package json would help to illustrate?

Reply:
I updated the message so it is more packed with all necessary information and steps.